pilgrimor/abc/engine.py

```diff
@@ -53,18 +53,35 @@ def execute_sql_with_return(
         """
 
     @abstractmethod
-    def execute_sql_no_return(
+    def execute_sql_with_no_return(
         self,
         sql_query: str,
         sql_query_params: Optional[List[Any]] = None,
         in_transaction: Optional[bool] = True,
     ) -> None:
         """
-        Executes sql query and do not return any output.
+        Executes sql query.
 
         By default query must be executed in transaction.
 
         :param sql_query: sql query to execute.
         :param sql_query_params: parameters for sql query.
         :param in_transaction: execute in transaction or not.
         """
+
+    @abstractmethod
+    def execute_version_migrations(
+        self,
+        version_migrations: List[Dict[str, str]],
+        sql_query_params: Optional[List[Any]] = None,
+        in_transaction: bool = True,
+    ) -> None:
+        """
+        Executes all migrations sql queries and do not return any output.
+
+        By default query must be executed in transaction.
+
+        :param version_migrations: sql queries dict by migrations.
```

Comment:
I don't this param explain is enought because it isn't clear what word `version` means in `version_migrations`. I don't understand)